src/quo2/components/navigation/bottom_nav_tab.cljs

```diff
@@ -0,0 +1,64 @@
+(ns quo2.components.navigation.bottom-nav-tab
+  (:require [quo.react-native :as rn]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.counter.counter :as counter]))
+
+(defn toggle-background-color [background-color press-out? pass-through?]
+  (let [color (cond
+                press-out?    nil
+                pass-through? colors/white-opa-5
+                :else         colors/neutral-70)]
+    (reset! background-color color)))
+
+(defn bottom-nav-tab
+  "[bottom-nav-tab opts]
+   opts
+   {:icon              :main-icons2/communities
+    :selected?         true/false
+    :notification?     true/false
+    :notification-type :unread/:counter
+    :counter-label     number
+    :on-press          bottom-tab on-press function
+    :pass-through?     true/false
+  "
+  [_]
+  (let [background-color (reagent/atom nil)]
+    (fn [{:keys [icon selected? notification? notification-type counter-label on-press pass-through?]}]
```

Comment:
lets discuss using `?` with the team

Reply:
yes, sounds good. Thank you

Reply:
so for now decision to keep using `?` only probably `notification?` could be renamed , up to you